### PR TITLE
Add remote-storage endpoint and path-style params

### PR DESCRIPTION
**Pull request by gaelreyrol**
## Purpose of this pull request

If the following https://github.com/magento/magento2/pull/32937 PR is accepted, those changes will be needed to improve remote storage parameters documentation.

## Affected DevDocs pages

-  https://devdocs.magento.com/guides/v2.4/config-guide/remote-storage/config-remote-storage.html#remote-storage-options

### CODE REVIEW

**Comment by magento-engcom-team** [comment]
@gaelreyrol thank you for contributing. Please accept Community Contributors team invitation [here](https://github.com/orgs/magento/invitation) to gain extended permissions for this repository.

**Comment by jeff-matthews** [comment]
@gaelreyrol, we ❤️ seeing dual code + doc contributions like this! We'll keep an eye on magento/magento2#32937 and process this if it gets accepted. Thank you!

File: src/guides/v2.4/config-guide/remote-storage/config-remote-storage.md [hunk 1]
```diff
@@ -54,12 +56,19 @@ The following examples enable the remote storage with an AWS S3 storage adapter
    bin/magento setup:config:set --remote-storage-driver="aws-s3" --remote-storage-bucket="myBucket" --remote-storage-region="us-east-1"
    ```
 
+-  Install new Magento with S3 compatible remote storage
+
+   ```bash
+   bin/magento setup:install --remote-storage-driver="aws-s3" -- remote-storage-endpoint="
+   ttp://localhost:9000" --remote-storage-bucket="myBucket" --remote-storage-region="us-east-1" --remote-storage-path-style=1
```
Comment (aleron75) [inline src/guides/v2.4/config-guide/remote-storage/config-remote-storage.md:63]: Shouldn't this be "http://..." or am I missing something?

Comment (gaelreyrol) [reply src/guides/v2.4/config-guide/remote-storage/config-remote-storage.md:63]: Absolutely

**Review by BarnyShergold — CHANGES_REQUESTED** [review]

**Pull request closed without merging**